[JS] September release notes (#1750)
Azure/azure-sdk: releases/2020-09/js.md

#### GA

- Azure Storage Blob Changefeed
- Azure Form Recognizer

#### Updates

- Core Libraries
- Azure Storage
- Azure Event Hubs

#### Preview

- Identity
- Azure Key Vault
- Azure Service Bus
- Azure Tables
- Azure Schema Registry
- Azure Text Analytics

## Installation Instructions

To install the packages, copy and paste the below into a terminal.

```bash

> $> npm install @azure/storage-blob-changefeed

Reviewer: If we're telling them to paste this we should probably get rid of the `$>` and is more of a bash-ism than a universal command prompt.

Author: I'm sorry to say this is the format we have at the moment :/ but I will send the feedback to the docs team.

> $> npm install @azure/storage-blob-changefeed

Reviewer: ```suggestion
npm install @azure/service-bus@next
$> npm install @azure/storage-blob-changefeed
```

Author: Is this a format fix or a fix to the package itself?

Reviewer: It's a format fix + it's missing the best package - @azure/service-bus :)

$> npm install @azure/storage-queue
$> npm install @azure/storage-file-share
$> npm install @azure/storage-datalake
$> npm install @azure/storage-blob
$> npm install @azure/ai-form-recognizer
$> npm install @azure/event-hubs
$> npm install @azure/identity@next
$> npm install @azure/keyvault-keys@next
$> npm install @azure/keyvault-secrets@next
$> npm install @azure/keyvault-certificates@next
$> npm install @azure/keyvault-admin@next
$> npm install @azure/data-tables@next

> $> npm install @azure/schema-registry@next

Reviewer 2: There is also `@azure/schema-registry-avro`

Author: thank you! I'll add it.

> $> npm install @azure/ai-text-analytics@next

Reviewer: Alphabetize? :)

Author: I wanted to go in the order of the updates above. I'll check what other languages are doing.

Author: We didn't push this alphabetized in the previous release. I'll leave it based on the previous list until this thread gets more feedback.

#### GA

> - Azure Storage Blob Changefeed

Reviewer 3: Oops, will sill be in preview.

- Azure Form Recognizer

#### Updates

- Core Libraries
- Azure Storage
- Azure Event Hubs

#### Preview

- Identity
- Azure Key Vault
- Azure Service Bus
- Azure Tables

> - Azure Schema Registry

Reviewer 4: Please remove all content on Schema Registry. Since this service will be publicly available only after Ignite, we have decided to not blog about it before the conference

> - Azure Text Analytics

Reviewer 5: ```suggestion
- Azure Text Analytics
- Azure Event Grid
```

## Installation Instructions

To install the packages, copy and paste the below into a terminal.

```bash
$> npm install @azure/ai-form-recognizer
$> npm install @azure/storage-queue
$> npm install @azure/storage-file-share

> $> npm install @azure/storage-datalake

Reviewer 3: ```suggestion
$> npm install @azure/storage-file-datalake
```

$> npm install @azure/storage-blob
$> npm install @azure/ai-form-recognizer

> $> npm install @azure/event-hubs

Reviewer 5: Adding the new `@azure/eventgrid` package to this list:

```suggestion
$> npm install @azure/event-hubs
$> npm install @azure/eventgrid@next
```

> Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:

Reviewer 6: I see that quick links in a couple of releases notes from late 2019, but not since.  Feel free to add it back, but no other languages have it.

> Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:

Reviewer 6: Removing for now

### Azure Form Recognizer

We are pleased to announce the general availability (GA) of the `@azure/ai-form-recognizer` package version 3.0.0.

#### Azure Form Recognizer [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md)

##### Features
- Train and utilize custom and/or predefined AI models to extract information from text document images.
- Uses the Azure Form Recognizer v2.0 (GA) endpoint for the latest service features.

> ### Azure Storage

Reviewer 4: @ljian3377, It looks like we have a copy of the changelog for the storage packages here. The idea of this release notes is to bring attention to [Developer impacting changes](https://github.com/Azure/azure-sdk/blob/d8c70b795c0c364d29d3c66bfb0e9d3ce163acef/docs/policies/releasenotes.md#whats-a-developer-impacting-change) and not really call out every feature added or every bug fixed.

Some specific feedback:

> - Supported RehydratePriority.
> - Supported Object Replication Service.
> - Supported Append Blob Seal.
> - Supported Tags conditional operations

When calling out the feature, we should add enough context/details. For example, `Supported RehydratePriority` does not tell an average storage user as to what new feature is the sentence talking about.

> Supported Quick Query. Added a new API `DataLakeFileClient.query()`.

A one line description of what a quick query is would be helpful

> The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628)

Use of `newPipeline` is considered an advanced scenario. The bug being fixed may not apply to most users and does not have a wide impact. I would recommend skipping this

> High level upload functions `BlockBlobClient.uploadFile()`, `BlockBlobClient.uploadStream()` and `BlockBlobClient.uploadBrowserData()` now support setting tier. Fixes bug [9062](https://github.com/Azure/azure-sdk-for-js/issues/9062).

This feels like a new feature and not a bug. Adding a few examples of tiers would be helpful.

> The Content-Length header is no longer ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

Instead of linking to the different bugs, can we add a sentence as to the scenarios where this bug fix matters to the user? Also, looks like this fix was made for all packages. Consider pulling it up before the individual packages are listed

> - Supported `abortSignal` when fetching Change Feed events.
> - Added new constructor overloads for the `BlobChangeFeedClient`.

Abort Signal support is not that much of a high impact, I would recommend skipping it.  For the constructor overloads, can you add a line as to what these overloads now allow users to do which was not possible before?

Reviewer 4: The above concerns have been addressed in https://github.com/Azure/azure-sdk/pull/1750/commits/4feb1d4dd6d1ab18a915833bdc34b04df768ffac. Thanks @ljian3377

      subQueue: "deadLetter"
    });
    ```

- Renames:
  - The `ServiceBusManagementClient` has been renamed to `ServiceBusAdministrationClient`. See
    [Issue 11012](https://github.com/Azure/azure-sdk-for-js/issues/11012) for more details.
  - All senders and receivers are now prefixed with `ServiceBus`: `ServiceBusSender`, `ServiceBusReceiver`, `ServiceBusSessionReceiver`
  - Lock duration fields for receivers have been renamed to apply to message locks and session locks:
    - `maxMessageAutoRenewLockDurationInMs` to `maxAutoRenewLockDurationInMs`
    - `autoRenewLockDurationInMs` -> `maxAutoRenewLockDurationInMs`
  - `SessionReceiver.{get,set}State` has been renamed to `SessionReceiver.{get,set}SessionState`
  - Administration API:
    - Property `defaultMessageTtl` renamed to `defaultMessageTimeToLive` (Wherever applicable)
    - `updatedAt` renamed to `modifiedAt`
    - `ServiceBusManagementClientOptions` for `ServiceBusManagementClient` is replaced by `PipelineOptions` from `@azure/core-http`
    - `AuthorizationRule.accessRights` type has been changed to be a string union with the available rights.

#### New Features in @azure/service-bus@7.0.0-preview.6
  

> - Support using the SharedAccessSignature from the connection string.

Reviewer 4: ```suggestion
- Support a new key-value pair in the connection string for the Shared Access Signature. The key to use is `SharedAccessSignature`.
```

  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
- Added a new field `amqpAnnotatedMessage` to the received message which will hold the received
  message in its raw form, complete with all parts of the message as per the [AMQP spec](https://www.amqp.org/sites/amqp.org/files/amqp.pdf).
- Added `ServiceBusAdministrationClient.ruleExists()`

> 

Reviewer 7: ```suggestion

- Options to create a queue and topic now support `enableExpress` boolean property. `enableExpress` indicates whether Express Entities are enabled on a queue or topic. An express queue holds a message in memory temporarily before writing it to persistent storage.
  [PR 10984](https://github.com/Azure/azure-sdk-for-js/pull/10984)
```

### Azure Tables

> We're releasing the first beta version of the new Azure Tables client, following our [guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) which ensures it is idiomatic, consistent, and diagnosable. It supports essential operations of the service, such as creating and deleting tables, as well as querying, creating, reading, updating and deleting entities.

Reviewer 4: I like this first part where we talk about guidelines. We should have something like this everytime we ship the first preview version for a service. 

@sadasant, Can you apply similar text for Event Grid?

#### Azure Tables [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/CHANGELOG.md)

> ##### New Features on @azure/data-tables@1.0.0-beta.1
> 
> - First beta release of this new package.
> - Provides client API to interact with the Azure Tables service
> - [Readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/README.md)

Reviewer 4: We can skip this entire section as nothing new is being shared when compared in the introduction a few lines above

```suggestion
```

Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:

---

=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===

Keep in mind that:

- Including the package name in the headers makes the URL links work for multiple packages.
- Only include customer-facing changes.
- The format of this file will be cleaned up once all of your proposals are in.

---

### _Package name_

#### _Package name_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)

(leave blank)

##### Breaking Changes on _Package name_

- _Add one or more, or remove the "Breaking Changes on ..." entire section._

##### New Features on _Package name_

- _Add one or more, or remove the "New Features on ..." section._

##### Major Fixes on _Package name_

- _Add one or more, or remove the "Major Fixes on ..." section._

## Latest Releases

> 

Reviewer: ```suggestion
### @azure/service-bus@7.0.0-preview.5

#### @azure/service-bus@7.0.0-preview.5 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/CHANGELOG.md)
 
##### Breaking Changes in @azure/service-bus@7.0.0-preview.5

- Renames
  - Property `defaultMessageTtl` -> `defaultMessageTimeToLive`(Wherever applicable)
  - `maxMessageAutoRenewLockDurationInMs` -> `maxAutoRenewLockDurationInMs`
  - `updatedAt` -> `modifiedAt`
  - Prefixing `Sender`, `Receiver`, `SessionReceiver` with `ServiceBus`
  - `autoRenewLockDurationInMs` -> `maxAutoRenewLockDurationInMs`
  - `{get,set}State` -> `{get,set}SessionState`(methods on session receiver)
- Add ruleExists API and tests
- `ServiceBusManagementClientOptions` for `ServiceBusManagementClient` is replaced by `PipelineOptions` from `@azure/core-http`
- `sessionLockedUntilUtc` is readonly and not undefined
  [PR 10625](https://github.com/Azure/azure-sdk-for-js/pull/10625)
 
##### New Features in @azure/service-bus@7.0.0-preview.5
  
- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
```

Author: Thank you! I'll follow up here probably next week. I want to focus on KV this Friday.

Reviewer 8: Please note that the changelog file has not been updated on master yet, you can find it [here](https://github.com/Azure/azure-sdk-for-js/blob/552534724f3c10a0fe3bceb4f59fe21bd0a60c62/sdk/textanalytics/ai-text-analytics/CHANGELOG.md).
```suggestion
### @azure/text-analytics@5.1.0-beta.1

#### @azure/text-analytics@5.1.0-beta.1 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/CHANGELOG.md)

##### New Features on @azure/text-analytics@5.1.0-beta.1

 We are now targeting the service's v3.1-preview.2 API as the default instead of v3.0.
- We now have added support for opinion mining feature of Sentiment Analysis. To get this support, pass `includeOpinionMining` as True when calling the `analyzeSentiment` endpoint.
- We have added the `recognizePiiEntities` API which returns entities containing personal information for a batch of documents and also returns the redacted text.
- The `length` and `offset` properties are added to the `SentenceSentiment`, `Entity`, `Match`, `PiiEntity` and `CategorizedEntity` interfaces to represent the length of the sentence's text and its offset from the start of the document. The unit of distance used is UTF-16 code points.
- `bingEntitySearchApiId` property is now returned for entities returned by `recognizeLinkedEntities` API and is added to the `LinkedEntity` interface. This property is to be used in conjunction with the Bing Entity Search API to fetch additional relevant information about the returned entity.

##### Major Fixes on @azure/text-analytics@5.1.0-beta.1

- The `text` property of the `SentenceSentiment` interface is no longer marked as optional because the service has always returned it. This interface is used exclusively as an output type so this change does not break existing code.
```

cc @willmtemple @ramya-rao-a @iscai-msft @maririos @mssfang .

    - `maxMessageAutoRenewLockDurationInMs` to `maxAutoRenewLockDurationInMs`
    - `autoRenewLockDurationInMs` -> `maxAutoRenewLockDurationInMs`
  - `SessionReceiver.{get,set}State` has been renamed to `SessionReceiver.{get,set}SessionState`
  - Administration API:
    - Property `defaultMessageTtl` renamed to `defaultMessageTimeToLive` (Wherever applicable)
    - `updatedAt` renamed to `modifiedAt`
    - `ServiceBusManagementClientOptions` for `ServiceBusManagementClient` is replaced by `PipelineOptions` from `@azure/core-http`
    - `AuthorizationRule.accessRights` type has been changed to be a string union with the available rights.

#### New Features in @azure/service-bus@7.0.0-preview.6
  
- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
- Added a new field `amqpAnnotatedMessage` to the received message which will hold the received
  message in its raw form, complete with all parts of the message as per the [AMQP spec](https://www.amqp.org/sites/amqp.org/files/amqp.pdf).
- Added `ServiceBusAdministrationClient.ruleExists()`

### Azure Tables

> We're releasing a new beta version for a client of Azure Tables. It supports the essential functionality of the service, including creating and deleting tables, as well as querying, creating, reading, updating and deleting entities.

Reviewer 9: ```suggestion
We're releasing the first beta version of the new Azure Tables client, following our [guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) which ensures it is idiomatic, consistent, and diagnosable. It supports essential operations of the service, such as creating and deleting tables, as well as querying, creating, reading, updating and deleting entities.
```

#### Azure Tables [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/CHANGELOG.md)

##### New Features on @azure/data-tables@1.0.0-beta.1

- First beta release of this new package.
- Provides client API to interact with the Azure Tables service
- [Readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/README.md)

> 

Reviewer 3: ```suggestion

### Azure Storage

#### Azure Storage Blob [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/CHANGELOG.md)

##### New Features on @azure/storage-blob@12.2.0

- Supported RehydratePriority.
- Supported Object Replication Service.
- Supported Append Blob Seal.
- Supported Tags conditional operations.
- The Static Website Service now supports a DefaultIndexDocumentPath for a global HTTP 200 route within the static website. You can get it by `BlobServiceClient.getProperties()` and set it via `blobServiceClient.setProperties()`.

##### Major Fixes on @azure/storage-blob@12.2.0

- The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
- High level upload functions `BlockBlobClient.uploadFile()`, `BlockBlobClient.uploadStream()` and `BlockBlobClient.uploadBrowserData()` now support setting tier. Fixes bug [9062](https://github.com/Azure/azure-sdk-for-js/issues/9062).
- The Content-Length header is no longer ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

#### Azure Data Lake Storage [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-file-datalake/CHANGELOG.md)

##### New Features on @azure/storage-file-datalake@12.1.0

- Supported Quick Query. Added a new API `DataLakeFileClient.query()`.

##### Major Fixes on @azure/storage-file-datalake@12.1.0

- The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
- The Content-Length header is no more ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

#### Azure Storage File Share [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-file-share/CHANGELOG.md)

##### New Features on @azure/storage-file-share@12.2.0

- Supported share soft delete. Added `undeleteShare` to `ShareServiceClient`. `listShares` now can return deleted shares. Note: share soft delete only take effect on accounts with share soft delete feature enabled.

##### Major Fixes on @azure/storage-file-share@12.2.0

- The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
- The Content-Length header is no more ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

#### Azure Storage Queue [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-queue/CHANGELOG.md)

##### Major Fixes on @azure/storage-queue@12.1.0

- The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
- The Content-Length header is no longer ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

#### Azure Storage Blob Change Feed [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob-changefeed/CHANGELOG.md)

##### New Features on @azure/storage-blob-change-feed@12.0.0-preview.2

- Supported `abortSignal` when fetching Change Feed events.
- Added new constructor overloads for the `BlobChangeFeedClient`.

```

#### New Features in @azure/service-bus@7.0.0-preview.6
  
- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).
- Added a new field `amqpAnnotatedMessage` to the received message which will hold the received
  message in its raw form, complete with all parts of the message as per the [AMQP spec](https://www.amqp.org/sites/amqp.org/files/amqp.pdf).
- Added `ServiceBusAdministrationClient.ruleExists()`

### Azure Tables

We're releasing a new beta version for a client of Azure Tables. It supports the essential functionality of the service, including creating and deleting tables, as well as querying, creating, reading, updating and deleting entities.

#### Azure Tables [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/CHANGELOG.md)

##### New Features on @azure/data-tables@1.0.0-beta.1

- First beta release of this new package.
- Provides client API to interact with the Azure Tables service
- [Readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/README.md)

> 

Reviewer 3: ```suggestion
```

#### Azure Storage Queue [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-queue/CHANGELOG.md)

##### Major Fixes on @azure/storage-queue@12.1.0

- The `credential` parameter of `newPipeline()` function is now optional. If not specified, `AnonymousCredential` is used. Fixes bug [9628](https://github.com/Azure/azure-sdk-for-js/issues/9628).
- The Content-Length header is no longer ignored. Fixes bugs [8903](https://github.com/Azure/azure-sdk-for-js/issues/8903), [9300](https://github.com/Azure/azure-sdk-for-js/issues/9300) and [10614](https://github.com/Azure/azure-sdk-for-js/issues/10614).

#### Azure Storage Blob Change Feed [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob-changefeed/CHANGELOG.md)

##### New Features on @azure/storage-blob-change-feed@12.0.0-preview.2

- Supported `abortSignal` when fetching Change Feed events.
- Added new constructor overloads for the `BlobChangeFeedClient`.

> 

Reviewer 3: ```suggestion
```

Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:

---

=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===

Keep in mind that:

- Including the package name in the headers makes the URL links work for multiple packages.
- Only include customer-facing changes.
- The format of this file will be cleaned up once all of your proposals are in.

---

### _Package name_

#### _Package name_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)

(leave blank)

##### Breaking Changes on _Package name_

- _Add one or more, or remove the "Breaking Changes on ..." entire section._

##### New Features on _Package name_

- _Add one or more, or remove the "New Features on ..." section._

##### Major Fixes on _Package name_

- _Add one or more, or remove the "Major Fixes on ..." section._

> ## Latest Releases

Reviewer: ```suggestion
## Latest Releases

### @azure/event-hubs

#### @azure/event-hubs [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/CHANGELOG.md)

#### New Features in @azure/event-hubs@5.3.0

- Adds `loadBalancingOptions` to the `EventHubConsumerClient` to add control around
  how aggressively the client claims partitions while load balancing.
  ([PR 9706](https://github.com/Azure/azure-sdk-for-js/pull/9706)).
- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).

#### @azure/event-hubs@5.3.0 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/CHANGELOG.md)

#### New Features in @azure/event-hubs@5.3.0

- Adds `loadBalancingOptions` to the `EventHubConsumerClient` to add control around
  how aggressively the client claims partitions while load balancing.
  ([PR 9706](https://github.com/Azure/azure-sdk-for-js/pull/9706)).
- Support using the SharedAccessSignature from the connection string.
  ([PR 10951](https://github.com/Azure/azure-sdk-for-js/pull/10951)).

```

> ## Latest Releases

Reviewer 5: For Event Grid

```suggestion
#### @azure/eventgrid@3.0.0-beta.1  [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventgrid/eventgrid/CHANGELOG.md)

(leave blank)

##### Breaking Changes on @azure/eventgrid@3.0.0-beta.1

- The Event Grid SDK has been updated to follow the new Azure SDK Design Guidelines.

##### New Features on @azure/eventgrid@3.0.0-beta.1

- The Event Grid SDK now supports sending and deseriazing events using the Cloud Events 1.0 schema.

```

> 

Reviewer 10: Still not exactly sure what to put in here for FR, but here's a stab:

```suggestion
### @azure/ai-form-recognizer@3.0.0

#### @azure/ai-form-recognizer@3.0.0 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md)

We are pleased to announce the general availability (GA) of the `@azure/ai-form-recognizer` package version 3.0.0.

##### Features
- Train and utilize custom and/or predefined AI models to extract information from text document images.
- Uses the Azure Form Recognizer v2.0 (GA) endpoint for the latest service features.
```
